docs/people-and-groups/saml-okta.md

```diff
@@ -71,42 +109,89 @@ You can click **Preview SAML assertion** to view the XML file generated by Okta.
 </saml2:Assertion>
 ```
 
-## Managing signing certificates in Okta
+## Set up SAML in Metabase
+
+Once you set up your SAML app in Okta, you'll need to configure SAML in Metabase. You'll need some information from Okta, so go to the page for the Metabase app integration in Okta, and in the **Sign On** tab, and click on **View SAML setup instructions**.
+
+Use the information from Okta SAML instructions to fill the Metabase SAML form in **Admin panel** > **Authentication** > **SAML**:
+
+| Metabase SAML                      | Okta SAML                            |
+| ---------------------------------- | ------------------------------------ |
+| SAML Identity Provider URL         | Identity Provider Single Sign-On URL |
+| SAML Identity Provider Certificate | X.509 Certificate\*                  |
+| SAML Identity Provider Issuer      | Identity Provider Issuer             |
+
+\*Make sure to include any header and footer comments, like `---BEGIN CERTIFICATE---` and `---END CERTIFICATE---`.
+
+## Configure group mappings
+
+You can configure Metabase to automatically assign users to Metabase groups when they log in. You'll need to create a SAML attribute statement that will pass the groups information to Metabase, and then configure Metabase to read this attribute and map its contents to Metabase groups.
+
+You can either [use a custom user profile attribute](#use-a-user-profile-attribute-to-assign-groups) that contains user's Metabase groups, or [leverage Okta User Groups](#map-okta-user-groups-to-metabase-groups).
+
+### Use a user profile attribute to assign groups
+
+You can create a custom user profile attribute and fill it with the Metabase groups for each user.
+
+1. In Okta **Profile Editor**, [create a new User Profile attribute](https://help.okta.com/en-us/content/topics/users-groups-profiles/usgp-add-custom-user-attributes.htm) called `metabaseGroups`, which can be a `string` or a `string array`.
+   ![New User Profile attribute](images/okta-new-attribute.png)
+2. For each user in Okta, fill the `metabaseGroups` attribute with their Metabase group(s).
+
+   ![Metabase groups attribute](images/okta-adding-groups.png)
+
+   We recommend that you use the same names for the groups in Okta as you would use in Metabase.
+
+   Keep in mind that Metabase groups don't have to correspond to Okta User Groups. If you'd like to leverage Okta User Groups to set up Metabase Groups, see [Map Okta User Groups to Metabase groups](#map-okta-user-groups-to-metabase-groups).
```

Comment:
```suggestion
   Metabase groups don't have to correspond to Okta User Groups. If you'd like to use Okta User Groups to set up Metabase Groups, see [Map Okta User Groups to Metabase groups](#map-okta-user-groups-to-metabase-groups).
```

Yes, "leverage" avoids the "use ... User" situation, but it's still pretty corporatese.